pkg/metadata/annotations.go

```diff
@@ -144,6 +144,16 @@ const (
 	// reconciler-manager reads it. If set to true, the reconciler-manager will
 	// create the reconciler with the Namespace controller in the reconciler container.
 	DynamicNSSelectorEnabledAnnotationKey = configsync.ConfigSyncPrefix + "dynamic-ns-selector-enabled"
+
+	// ImageURLAnnotationKey is the annotation key applied to R*Sync objects
```

Comment:
Similarly,

```
// SourceURLAnnotationKey stores the source URL where the source commit is fetched from.
// It is only set for OCI and Helm source, and unset for the Git source.
// It can be used for OCI signature verification.
```

Reply:
Done